docs/testing/debug-with-vs-extension.md

```diff
@@ -62,7 +62,7 @@ Once Visual Studio Code is open, navigate manually to the project folder.
 }
 ```
 
-7. In the section of JSON you just copied, find the "url" section. In this URL, you will need to replace the uppercase HOST text with the host application for your Office add-in. For example, if your Office add-in is for Excel, your URL value would be "https://localhost:3000/taskpane.html?_host_Info=<strong>Excel</strong>$Win32$16.01$en-US$\$\$\$0".
+7. In the section of JSON you just copied, find the "url" section. In this URL, you will need to replace the uppercase HOST text with the application that is hosting your Office add-in. For example, if your Office add-in is for Excel, your URL value would be "https://localhost:3000/taskpane.html?_host_Info=<strong>Excel</strong>$Win32$16.01$en-US$\$\$\$0".
```

Comment:
This may be a change for another time but I'm seeing "Office Add-in" and "Office add-in" being used...

Reply:
I'll fix "Office add-ins" -> "Office Add-ins". I think we should change the singular too, but I wonder why it wasn't already like that (there's many instances of this).